Tasks/CmdLineV2/task.json

```diff
@@ -59,6 +59,15 @@
             "required": false,
             "helpMarkDown": "If this is true, this task will fail if any errors are written to the StandardError stream.",
             "groupName": "advanced"
+        },
+        {
+            "name": "ignoreHostExpression",
+            "type": "boolean",
+            "label": "Ignore HostException from Invoke-Expression cmdlet (works only on windows)",
+            "defaultValue": "true",
+            "required": false,
+            "helpMarkDown": "PowerShell version of the task invokes script through Invoke-Expression cmdlet which has a bug, that could cause unexpected HostException in some situations. While this is still the problem on the PowerShell side, this option allows mitigating the issue.",
```

Comment:
It probably will depend on Powershell version, and this message could confuse end customer - since task is just a black box for pipelines user. Could it make sense to rewrite it a little bit from the customer's point? (for example, when it is useful to turn it on)

Reply:
Updated message